includes/apple-exporter/class-exporter.php

```diff
@@ -357,7 +330,7 @@ private function content_title() {
 	 *
 	 * @access private
 	 */
-	private function prepare_for_encoding( &$data ) {
+	public function prepare_for_encoding( &$data ) {
```

Comment:
```suggestion
	public static function prepare_for_encoding( &$data ) {
```

Reply:
Remember to change references to this function in this file from `$this->prepare_for_encoding` to `self::prepare_for_encoding`.